platform/macos/export/export.cpp

```diff
@@ -45,6 +45,8 @@ void register_macos_exporter() {
 #else
 	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/macos/rcodesign", PROPERTY_HINT_GLOBAL_FILE));
 #endif
+	EDITOR_DEF_BASIC("export/macos/actool", "");
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/macos/actool", PROPERTY_HINT_GLOBAL_FILE));
```

Comment:
I noticed these editor settings are not present in the class reference XML. This is not an issue specific to this PR (it applies to other export editor settings), but we should look into addressing this eventually.

Reply:
This seems to be true for all editor settings in the export plugins.